src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java

```diff
@@ -191,4 +192,14 @@ public void split_field() throws Exception {
                 TestsConstants.TEST_INDEX + "/account where address is not null " +
                 " limit 10  ";
     }
+
+    @Test
+    public void functionAssign() throws Exception {
+        String query = "SELECT 10 as key "+
+                "from " + TEST_INDEX_ACCOUNT + "/account limit 1";
+
+        assertThat(
+                executeQuery(query),
+                hitAny(kvInt("/fields/key/0", equalTo(10))));
+    }
 }
```

Comment:
please add another test without assigning an alias to the field.

Reply:
Done.